### PR TITLE
Add page layouts for patterns, principles and standards

### PR DESCRIPTION
**Pull request by keithkennedyHO**
- Added layouts for:
	- patterns
	- principles
	- standards
- New layouts inherit from common `content` layout.
  - Content pages have been updated to use the appropriate layout and remove tag/date code, cleaning them up.
- Improved patterns, principles and standards pages. These now show a paginated list of all pages related to them
- Added descriptions of patterns, principles and standards, which display on the front page and the collection page

# Code change
I can confirm:
## Accessibility considerations
- [x] This change might impact accessibility, automated aXe tests cover the impact

# Content change 
I can confirm:
- [x] Content is suitable to open source, i.e.:
    - Content does not relate to unreleased gov policy
    - Content does not refer to anti-fraud mechanisms
    - Content does not include sensitive business logic

### CODE REVIEW

**Review by edhamiltonHO — APPROVED** [review]
LGTM

**Review by jeff-horton-ho-sas — COMMENTED** [review]
Now that the LHS navigation is gone there is no easy way to navigate around the site. We either need to restore the LHS navigation or reintroduce breadcrumbs.

Can the standards and prinicpals pages have a related links section on the right with links to the relevant standard?

Additionally we should create a ticket to add descriptions to each of the standards, principles, patterns so that they show in the lists.

File: .eleventy.js [hunk 1]
```diff
@@ -69,6 +69,31 @@ module.exports = function(eleventyConfig) {
...
+    eleventyConfig.addCollection("homepageLinks", function(collectionApi) {
+      return collectionApi.getFilteredByGlob([ 
+        "**/patterns.md",
+        "**/principles.md",
+        "**/standards.md"]);
+    });
+
+    eleventyConfig.addCollection("getAllStandards", function(collectionApi) {
+      return collectionApi.getFilteredByGlob("**/standards/*.md").sort(function(a, b) {
+          return a.data.id.localeCompare(b.data.id); // sort by ID ascending
```
Comment (jeff-horton-ho-sas) [inline .eleventy.js:81]:
```suggestion
          return a.data.id.localeCompare(b.data.title); // sort by title ascending
```
Are we expecting most people to reference standards by ID? Think this list will be easier to browse alphabetically but happy to be challenged on that.

Comment (keithkennedyHO) [reply .eleventy.js:81]: I am unsure how most will reference standards. However, we are showing standards with "ID - Title" so based on that I think not ordering by ID may be confusing (doesn't look right to me, personally). My preference is ID ordering here, unless we changed the display so it did not start with ID.

Comment (jeff-horton-ho-sas) [reply .eleventy.js:81]: Screenshot 2023-07-28 at 15 39 09
Screenshot 2023-07-28 at 15 38 58

I think the most prominent in both cases for me is the title, even if the ID is first. I found it jarring coming to the standards collection page from the other two that were in title order.

Comment (keithkennedyHO) [reply .eleventy.js:81]: Ah, I was referring to the sidebar links:
image

Agree it is not the best on the standards page itself. These both use the same collection, but we can make two collections for standards each with ordering on ID and title and use them both.

Comment (edhamiltonHO) [reply .eleventy.js:81]: I was actually going to suggest we sort by ID descending..

I think ultimately we will want to create a way people can browse the standards by tag, maybe with a good filter. So happy for us to leave as-is for now

**Review by jeff-horton-ho-sas — APPROVED** [review]

**Pull request merged**